### ISSUE TITLE
TheTVDB has deleted Gundam SEED

### ISSUE THREAD

**Issue by atraholt** [ScudLee/anime-lists#303]
And that's kind of a big deal, because that [entry](https://www.thetvdb.com/dereferrer/series/74474/allseasons) had a *lot* of specials tied to it. Replacement [entry](https://thetvdb.com/?tab=seasonall&id=367683) has zero. Since I'm a big weirdo that caches everything locally, I have a copy of the old entry as XML, and I was hoping we could try and find where those specials have gone now and maybe add them back. [Trakt](https://trakt.tv/shows/mobile-suit-gundam-seed) also has a visual copy of it (for now)
[74474.xml.txt](https://github.com/ScudLee/anime-lists/files/3832543/74474.xml.txt)

**Comment by purposelycryptic** [comment]
Good catch!
I just checked TVDB, and it looks like in the chaos of the v3 API change, SEED has now been split into SEED, SEED Destiny, SEED Remaster, SEED Destiny Remaster, Stargazer OVA (as a TV series), Stargazer OVA (as a movie), and each of the special edition compilation movies as individual movie entries (the other specials, like MSV Astray and Between the Stars, are attached to SEED and SEED Destiny as specials).

It's kind of a huge mess, because Stargazer is *also* attached to a series as a special (in addition to being listed as a series, and as a movie), and the same is true for the Special Edition movies - LOTS of duplication all around. It does seem like everything has an entry again, but almost everything from SEED now actually has two, which... \*facepalm\*

Being seemingly sick forever at this point, I hadn't recently looked into the anime-lists or TheTVDB, at least not since the *Big Change*, which seems to have been way bigger than I imagined; I mean, they have a movie category now! Even mentioning the word used to be a capital offense.

I have family visiting for the next week, but if no one else has redone the mappings by the time they leave, I'll put it on my to-do list along with what remains of Winter 2020, and try to get everything done in a couple days.
